h/static/scripts/login-forms/config.ts

```diff
@@ -87,6 +87,7 @@ export type AccountSettingsConfigObject = ConfigBase & {
     'oidc.connect.google'?: string;
     'oidc.connect.facebook'?: string;
     'oidc.connect.orcid'?: string;
+    identity_delete?: string;
```

Comment:
This is marked as optional but it looks like you always set it in the backend?

Reply:
Yep, fixed